monitoring/src/main/scala/akkaviz/events/ActorSystems.scala

```diff
@@ -0,0 +1,21 @@
+package akkaviz.events
+
+import akka.actor.ActorSystem
+
+import scala.ref.WeakReference
+
+object ActorSystems {
+
+  private val systemReferences = scala.collection.mutable.Map[String, WeakReference[ActorSystem]]()
+
+  def systems: scala.collection.immutable.Map[String, ActorSystem] = systemReferences.flatMap {
+    case (name, ref) => ref.get.map {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2110675929/issues/source?bid=3121282&fileBranchId=3144103#l12)